### PR TITLE
Upgrade public suffix to elixir 1.3.

### PR DESCRIPTION
**Pull request by myronmarston**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,5 +3,7 @@ language: elixir
 otp_release:
   - 18.1
 elixir:
-  - 1.2.4
+  - 1.2.6
+  - 1.3.0
+before_install: "mix local.rebar --force"
```
Comment (benkirzhner) [inline .travis.yml]: Why is this line needed?

Comment (myronmarston) [inline .travis.yml]: Without it, the build stalls because mix prompts you to install rebar to continue compiling dependencies, but there is no one to confirm the prompt.

I had to do the same thing in Delorean to get the build on 1.3 to not stall at that stage.

**Comment by benkirzhner** [comment]
LGTM
